packages/ipfs/src/core/runtime/libp2p-browser.js

```diff
@@ -1,5 +1,6 @@
 'use strict'
 
+const Bootstrap = require('libp2p-bootstrap')
```

Comment:
I noticed that Bootstrap is only required in `components/libp2p`. Is this a bundle optimization?

Reply:
cc @achingbrain

Reply:
AFAIK the inline requires are to speed up node start-up time.

Reply:
ok, so I will add a comment for that and keep the Bootstrap as it was and only use the tag for the other one

Reply:
iirc the inline requires were done to reduce the cli execution time, not startup time (ie: if it's not needed for interacting with the running daemon, avoid requiring it). Not sure if this is still a problem with how things are setup internally.

Reply:
Should I change the comment to: `Requiring bootstrap inline in components/libp2p to reduce the cli execution time`?

Reply:
I don't think that's really accurate though. In reality the whole libp2p component should be ignored in CLI, as I don't think it's used at all (unless I'm mistaken). I'd leave it for now, and we can clean that all up when we update libp2p configuration.